pilot/pkg/proxy/envoy/v1/testdata/gateway-h2-listener.json.golden

```diff
@@ -0,0 +1,160 @@
+{
```

Comment:
someone actually trying out these golden files would be nice - I believe @ZackButcher was going to take a look.